Detectors/TPC/qc/include/TPCQC/Helpers.h

```diff
@@ -49,6 +51,8 @@ void setStyleHistogram2D(TH2& histo);
 /// Set nice style for vector of 2D histograms
 void setStyleHistogram2D(std::vector<TH2F>& histos);
 
+bool newZSCalib(const o2::tpc::CalDet<float>* refPedestal, const o2::tpc::CalDet<float>* refNoise, const o2::tpc::CalDet<float>* pedestal);
```

Comment:
```suggestion
bool newZSCalib(const o2::tpc::CalDet<float>& refPedestal, const o2::tpc::CalDet<float>& refNoise, const o2::tpc::CalDet<float>& pedestal);
```
you anyhow assume they are not nullpointers...